utils/config.go

```diff
@@ -247,6 +247,16 @@ func ReadConfig(r io.Reader, allowEnvironmentOverrides bool) (*model.Config, map
 
 	var config model.Config
 	unmarshalErr := v.Unmarshal(&config)
+	// https://github.com/spf13/viper/issues/324
```

Comment:
This restores a change @hmhealey recently reverted, but applies it at a lower level: just to the `Plugins` and `PluginStates` subkeys, not the rest of the configuration.